### ISSUE TITLE
Is it possible to add fingerprint support?

### ISSUE THREAD

**Issue by GhostFantik** [nekr0z/linux-on-huawei-matebook-13-2019#11]

**Comment by nekr0z** [comment]
Is it possible to add it to this Readme? Yes, sure. If you have the instructions on how to make this fingerprint sensor work in Linux, you can make a pull request, or just post them here and I add them.

Where to find those instructions? I don't know (or else I would have added them). Is it even possible to make this sensor work on Linux? I don't know (or else I would have written about it).

**Comment by GhostFantik** [comment]
OK! Thank you for answer

**Comment by ghost** [comment]
There's recently a Linux version(deepin) of Matebook 14, which also has a fingerprint scanner. Maybe someone can port the driver for other Matebook devices.

**Comment by GhostFantik** [comment]
Yeah. If I'm not mistaken, the version for matebook 13 will also be

**Comment by aetbaev** [comment]
Must be great!

**Comment by shostakalex** [comment]
Thank you for your work! Any movements of matebook 13 fingerprint here?
